spec/features/user_profiles_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'helpers'
+
+feature 'user can view profile' do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'users can view their profile' do
+    sign_in_as(user)
+    make_review_for_game(game)
+    visit users_path
+    within(".column") do
+      click_link user.username
+    end
+    expect(page).to have_content(user.username)
+    expect(page).to have_content("review of #{game.name}")
+  end
```

Comment:
I'm not sure I follow. Are you saying we don't need to actually write out the test? Or just that we could simplify it? Or something else?